pkg/config/config.go

```diff
@@ -129,9 +128,6 @@ func (c Config) MarshalYAML() (interface{}, error) {
 	enc.SetHook(func(in interface{}) (ok bool, out interface{}, err error) {
 		// Obscure the password fields for known types that do not obscure passwords.
 		switch v := in.(type) {
-		case etcd.Config:
-			v.Password = "<secret>"
-			return true, v, nil
```

Comment:
etcd.Config no longer needs an override (and has changed types; it's no longer a string) so we can remove this.